packages/sitecore-jss-nextjs/src/components/RichText.tsx

```diff
@@ -0,0 +1,94 @@
+import React, {useEffect, useRef} from 'react';
+import { useRouter } from 'next/router';
+import { RichText as ReactRichText } from '@sitecore-jss/sitecore-jss-react';
+
+export interface RichTextProps {
+	/** The rich text field data. */
+	field?: {
+			value?: string;
+			editable?: string;
+	};
+	/**
+	 * The HTML element that will wrap the contents of the field.
+	 * @default <div />
+	 */
+	tag?: string;
+	/**
+	 * Can be used to explicitly disable inline editing.
+	 * If true and `field.editable` has a value, then `field.editable` will be processed and rendered as component output. If false, `field.editable` value will be ignored and not rendered.
+	 * @default true
+	 */
+	editable?: boolean;
+
+	/**
+	 * Selector which should be used in order to prefetch it and attack event listeners
+	 */
+	internalLinksSelector?: string;
+
+	/**
+	 * Should component prefetch routes
+	 */
+	prefetch?: boolean;
+
+	/**
+	 * Locale which will be used when prefetch route
+	 */
+	locale?: string;
```

Comment:
Same here - can probably get rid of `locale` option, and also wondering if `prefetch` is necessary - why else would they be using this component?

Reply:
Removed `prefetch`, added `locale={false}`
https://nextjs.org/docs/advanced-features/i18n-routing#transition-between-locales